### PR TITLE
Issue 34.jb.1

### PR DESCRIPTION
**Pull request by jbutcher21**
#34 plus additions to capture search entity ID if search record was actually loaded.

### CODE REVIEW

File: G2Search.py [hunk 1]
```diff
@@ -236,12 +244,27 @@ def process_search(rowData, mappingDoc, g2Engine, searchFlags):
...
+
+    #--get target record from truthset
+    if truthFileName:
+        try:
+            cluster_id = truthsetRecords[rowData['DATA_SOURCE']][rowData['RECORD_ID']]
+            targetRecord = truthsetTargets[cluster_id]
+            rowData['TARGET_DATA_SOURCE'] = targetRecord['DATA_SOURCE']
+            rowData['TARGET_RECORD_ID'] = targetRecord['RECORD_ID']
+        except:
+            pass
```
Comment (antaenc) [inline G2Search.py:257]: This is heavily frowned upon, use instead: 

with suppress(Exception):
            cluster_id = truthsetRecords[rowData['DATA_SOURCE']][rowData['RECORD_ID']]
            targetRecord = truthsetTargets[cluster_id]
            rowData['TARGET_DATA_SOURCE'] = targetRecord['DATA_SOURCE']
            rowData['TARGET_RECORD_ID'] = targetRecord['RECORD_ID']

**Review by antaenc — APPROVED** [review]
I've made a couple of comments

File: G2Search.py (hunk 1 above)
Comment (jbutcher21) [reply G2Search.py:257]: ok, commented that code out.

File: README.md [hunk 2]
```diff
@@ -99,6 +99,8 @@ as indicated by the "source" attribute.
 - input: This is helpful so that you can quickly see what record was searched for.  
 You can specify any of the following ...
     - ROW_ID: the row number in the json file
+    - SEARCH_ENTITY_ID: the entity_id assigned to the search record if the search record was actually loaded
```
[the remark this answers is not in the document]
Comment (jbutcher21) [reply README.md:102]: I think "actually" is necessary

**Pull request merged**